src/rest_vol_group.c

```diff
@@ -349,6 +378,25 @@ RV_group_open(void *obj, const H5VL_loc_params_t *loc_params, const char *name,
     printf("-> Found group by given path\n\n");
 #endif
 
+    obj_type = H5I_UNINIT;
+
+    search_ret = RV_find_object_by_path(parent, name, &obj_type, RV_copy_creation_properties_callback, NULL,
+                                        &base64_binary_gcpl);
```

Comment:
You might want to consider having a combined `RV_copy_object_URI_and_domain_callback` and `RV_copy_creation_properties_callback` in the future so we aren't making multiple calls to RV_find_object_by_path for the same object.

Reply:
I'm a little wary of this - we already have distinct `RV_copy_object_URI_callback` and `RV_copy_object_URI_and_domain_callback` functions, so `RV_copy_URI_domain_creation_properties` would bring us up to three functions capturing a different combination of data, and it would only be used in this one place. Would one function that tries to get a wide selection of fields, but fails gracefully for ones that aren't defined be a decent solution?

Reply:
I agree, but it's a product of the architecture needing to change as functionality is added. It might make the client side code more complex, but I'd say that's probably dwarfed by the need to make at least one extra network request in this case or possibly more if an optimized RV_find_object_by_path can't be used for whatever reason.

Experimenting with a single function to grab everything needed is reasonable. You could also have a composition of callback functions as was done for the `RV_copy_object_URI_and_domain_callback` callback, but the issue with that is you end up with several layers recreating parse trees for the same response buffer just to grab a single field out of it.